captum/optim/_utils/image/common.py

```diff
@@ -105,30 +161,38 @@ def weights_to_heatmap_2d(
     assert weight.dim() == 2
     assert len(colors) == 5
 
-    def get_color(x: str) -> torch.Tensor:
+    def get_color(x: str, device: Union[str, torch.device] = "cpu") -> torch.Tensor:
```

Comment:
`x.device` should always return a torch.device variable and users aren't able to change that, so I don't think we need to have the `str` type hint here.

```
... device: torch.device = torch.device("cpu")) -> torch.Tensor:
```

Reply:
Fair fair, I'll remove the str and change the default to torch.device('cpu')